src/database/static_methods.rs

```diff
@@ -358,6 +358,16 @@ impl Database {
             NO_PARAMS,
         )?;
 
+        conn.execute(
+            "CREATE INDEX IF NOT EXISTS room_events_by_timestamp ON events (room_id, server_ts DESC, event_id)",
+            NO_PARAMS,
+        )?;
+
+        conn.execute(
+            "CREATE INDEX IF NOT EXISTS room_event_id ON events (event_id)",
```

Comment:
Is the index meant to have `room_id` as well...? If not, why is `room` in the name?

Reply:
No, I guess I just meant `room events` as opposed to `to-device events`, but we're not dealing with to-device events here at all.

A professional deformation from dealing with the crypto side of things in the rust-sdk for too long. :smile: